### PR TITLE
Add linters

### PR DESCRIPTION
**Pull request by AndreLau11**
Add the website project, including all elements on it:

- The main page is which gives you a general panorama of what the page is about.
- The main program section with multiple available actions allows you to select what you would like to do on the page.
- The featured speaker's sections lead to the information of expositors that will be on the program.
- The partner's area gives you a detail of those who will be taking during the exposition.
- The about me section gives you more detail of the companies information and logotypes and selection of the last one.
- The past events that had space during the last years

On the code, it was implemented links and the speaker area were implemented dynamically with javascript, and also some of the links are working correctly according to the available sections.

Video Description:

https://www.loom.com/share/29f331244147441da8c93eeb734f7fe2

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,483 @@
+body {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  background-image: url(./images/first_page/img.jpg);
+}
```
Comment (ReshyResh) [inline styles.css]: - [x] You're currently using the background image for  the whole body on both the index and the about page, but as you can see from the template, only the main section uses that background, please make sure to change it.

File: README.md [hunk 2]
```diff
@@ -2,18 +2,19 @@
 
 # Project Name
```
Comment (ReshyResh) [inline README.md:3]: - [x] Please make sure to either remove or fill the empty sections of the readme, and I suggest adding a screenshot of the application to make it look more professional.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+                <li><img src="./images/first_page/Rectangle.png" alt=""></li>
+            </ul>
+        </div>
+        <div class="mainpage container">
+            <h3 class="greetings">"Hello, Sharing World"</h3>
+            <h1 class="main">Creating Global Video Forums Regarding The News On Videogames</h1>
+            <p class="mainparagraph">An awesome reunion of videogame news sharing the information on them and how they can change your life on prices and by dedicating yourself to do so, how much are this companies given as rewards and what you need to do in order to attempt to be the new bigest gamer of all the times.</p>
+            <h2 class="mainsubtitl">2022.03.14(Monday) ~ 18(Friday)</h2>
+            <p class="mainparagraph2">@Play Station, Nintendo, Epic Game and more.</p>
+        </div>
```
Comment (ReshyResh) [inline index.html:75]: - [x] Your main section's width is too large compared to the template. It shouldn't be this wide, but rather stop a bit after the half. Also, please make sure that you're matching the template in terms of height, your main section is really tiny compared to the template, it should take up the whole width of the screen.

| Your website  | Template |
| ------------- | ------------- |
|   |   |

File: main.html [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+            <p>The logo of Creating Global Video Forums was decided through the logo competition from the first of February to the 28th of the same month. After online public voting and selection commmittee's judging process, the logo from Naresh Agrawal won the prize. After consultation with Mr. Agrawal we optimized the prize-winning-logo fro this year's summit as following.</p>
+            <div class="companiesLogo3">
+                <a href="index.html"><img class="logoIcon" src="./images/first_page/logo.jpg" alt=""></a>
+                <p>Global Video Games</p>
+            </div>
+        </div>
+        <div class="past container">
+            <h4>See the past Creating Global Video Forums</h4>
+            <p>Take a loog at the previous works we have done so far and search for more inforamtion about how this events ara alike.</p>
+            <div>
```
Comment (ReshyResh) [inline main.html]: - [x] This section is currently missing the margins and paddings as shown in the template, please make sure to differentiate the different sections to better match the template. Make sure to also remove the background image from the sections that shouldn't have it.


| Your website  | Template |
| ------------- | ------------- |
|   |   |

**Review by ReshyResh — CHANGES_REQUESTED** [review]
# Hi @AndreLau11 , 

### Good job so far with your capstone!
  - ✔ Design looks similar to the template.
  - ✔ Used correct semantic tags and made the website accessible.
  - ✔ No linter errors.


#### There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

- [x] Please make sure to add a video recording of yourself presenting the project and showcasing a feature you're most proud of, as per requirement.
- [x] Your pull request description should be a list of the changes in the repository, it's currently very short and doesn't reflect what has been done in the project, please make sure to add a more descriptive one.

_Please check the comments under the review for more messages._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [x] [OPTIONAL] For your next projects, I suggest you make your commit messages more detailed, it will make it easier to find eventual bugs arising between each commit.


Cheers and Happy coding! 👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (AndreLau11) [reply styles.css]: I could check that both uses the background image, however just for the initial area.

**Review by ReshyResh — APPROVED** [review]
# Hi @AndreLau11 , 

### Good job addressing the changes requested previously!
#### Your capstone is complete!  There is nothing else to say other than... it's time to merge it!
Congratulations and good luck with your next module! 🎉 


Cheers and Happy coding!👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**